ssm/ssm2.py

```diff
@@ -112,6 +112,12 @@ def __init__(self, hosts_and_ports, qpath, cert, key, dest=None, listen=None,
             log.info('Messages will be encrypted using %s', enc_cert)
             if not os.path.isfile(self._enc_cert):
                 raise Ssm2Exception('Specified certificate file does not exist: %s.' % self._enc_cert)
+            # Check that the encyption certificate has not expired.
+            if not crypto.verify_cert_date(enc_cert):
+                raise Ssm2Exception(
+                    'Encryption certificate %s has expired. Please obtain the '
+                    'current one for the server you are sending to.' % enc_cert
```

Comment:
Maybe change this to `Please obtain the current one for the accounting repository you are sending to`?

Don't want people confusing the 'server' for the broker

Reply:
Changed somewhat, though I've also made it less accounting specific.